### ISSUE TITLE
Suggestion: Use mcollective instead of capistrano

### ISSUE THREAD

**Issue by raphink** [pzim/reaktor#16]
Using [Zack's r10k MCollective](https://forge.puppetlabs.com/zack/r10k#mcollective-support) agent would allow to contact all Puppetmasters without having to list them.

**Comment by shawn-sterling** [comment]
Or make it plugable so people can use other whatever agent they want. Some large puppet deployments use ansible for this function.

**Comment by pzim** [comment]
Agreed - making this bit pluggable would definitely be the way to go. If there is enough interest in this, I can start looking at implementing a pluggable interface for it.

**Comment by ptierno** [comment]
:+1:
